development/containers/php/Dockerfile

```
@@ -59,6 +64,9 @@ ENTRYPOINT ["make", "php-cs-fixer-check"]
 
 FROM php-base-with-composer AS develop
 
-RUN pecl install xdebug
+RUN \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)